STANDARDS.md

```diff
@@ -115,14 +111,17 @@ Pre-submission filtering:
 
 Post-submission filtering:
 
-- OpenFF toolkit ingestion with strict stereo checking in RDKit
-- Hydrogen bonding
+- Stereochemistry is preserved (e.g. QCSubmit's `UnperceivableStereoFilter`)
+- CMILES (topology) change (e.g. QCSubmit's `ConnectivityFilter`)
+- Calculation ran successfully (e.g. QCSubmit's `RecordStatusFilter`)
+- Hydrogen bonding (for torsion drives only) (e.g. QCSubmit's `HydrogenBondFilter`)
```

Comment:
It might be nice to link to the code or, likely preferably, the docs for these filters.